Makefile

```diff
@@ -4,13 +4,9 @@ SWIFT_BUILD_FLAGS := -c release --disable-sandbox
 TOOL_NAME := swift-mod
 XCODE_DEFAULT_TOOLCHAIN := /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain
 GITHUB_REPO := ra1028/$(TOOL_NAME)
-DOCKER_IMAGE_NAME := swift:5.5
+DOCKER_IMAGE_NAME := swift:5.9
 
 ifeq ($(shell uname), Darwin)
-USE_SWIFT_STATIC_STDLIB := $(shell test -d $$(dirname $$(xcrun --find swift))/../lib/swift_static/macosx && echo use_swift_static_stdlib_flag)
-ifeq ($(USE_SWIFT_STATIC_STDLIB), use_swift_static_stdlib_flag)
-SWIFT_BUILD_FLAGS += -Xswiftc -static-stdlib
-endif
```

Comment:
We cannot pass `-static-stdlib` under the Darwin env anymore.
Rev: https://github.com/apple/swift-driver/pull/184